pkg/relabel/relabel_test.go

```diff
@@ -233,6 +233,54 @@ func TestRelabel(t *testing.T) {
 				"d": "976",
 			}),
 		},
+		{
+			input: labels.FromMap(map[string]string{
+				"a": "foo",
+				"b": "foo-bar",
+				"c": "bar",
+				"d": "baz",
```

Comment:
Sorry!! I did not pay attention, I was thinkin that break was breaking a loop :) Not I see it is a switch. I am happy to keep only "a" and "b:"

Thanks again!!!

Reply:
No worry at all